test/e2e/testnet.go

```diff
@@ -56,19 +59,157 @@ func (t *Testnet) CreateGenesisNode(version string, selfDelegation, upgradeHeigh
 	return nil
 }
 
-func (t *Testnet) CreateGenesisNodes(num int, version string, selfDelegation, upgradeHeight int64) error {
+func (t *Testnet) CreateGenesisNodes(num int, version string, selfDelegation, upgradeHeight int64, resources Resources) error {
 	for i := 0; i < num; i++ {
-		if err := t.CreateGenesisNode(version, selfDelegation, upgradeHeight); err != nil {
+		if err := t.CreateGenesisNode(version, selfDelegation, upgradeHeight, resources); err != nil {
 			return err
 		}
 	}
 	return nil
 }
 
-func (t *Testnet) CreateNode(version string, startHeight, upgradeHeight int64) error {
+func (t *Testnet) CreateAndSetupTxSimNodes(version string,
+	seed int,
+	sequences int,
+	blobRange string,
+	pollTime int,
+	resources Resources,
+	grpcEndpoints []string,
+) error {
+	for i, grpcEndpoint := range grpcEndpoints {
+		name := fmt.Sprintf("txsim%d", i)
+		err := t.CreateAndSetupTxSimNode(name, version, seed, sequences,
+			blobRange, pollTime, resources, grpcEndpoint)
+		log.Info().
+			Str("name", name).
+			Str("grpc endpoint", grpcEndpoint).
+			Msg("txsim created")
+
+		if err != nil {
+			log.Err(err).Str("name", name).
+				Str("grpc endpoint", grpcEndpoint).
+				Msg("txsim creation failed")
+			return err
+		}
+	}
+	return nil
+}
+
+// CreateAndSetupTxSimNode creates a txsim node and sets it up
+// name: name of the txsim knuu instance
+// version: version of the txsim docker image to be pulled from the registry
+// specified by txsimDockerSrcURL
+// seed: seed for the txsim
+// sequences: number of sequences to be run by the txsim
+// blobRange: range of blob sizes to be used by the txsim in bytes
+// pollTime: time in seconds between each sequence
+// resources: resources to be allocated to the txsim
+// grpcEndpoint: grpc endpoint of the node to which the txsim will connect and send transactions
+func (t *Testnet) CreateAndSetupTxSimNode(name,
```

Comment:
Yes, It can be accomplished by utilizing `CreateAndSetupTxSimNodes`(going to be renamed to `CreateTxClients`), and assigning the `grpcEndpoint` parameter with the gRPC endpoint of the same node multiple times. This function generates one tx client for every gRPC endpoint specified.